feature/identity/src/test/java/com/revenuecat/purchases/identity/IdentityManagerTests.kt

```diff
@@ -337,6 +384,20 @@ class IdentityManagerTests {
         assertCorrectlyIdentifiedWithAnonymous()
     }
 
+    @Test
+    fun `logOut resets offline customer info cache`() {
+        val identifiedUserID = "Waldo"
+        mockIdentifiedUser(identifiedUserID)
+        mockSubscriberAttributesManagerSynchronize(identifiedUserID)
+        every { mockDeviceCache.cleanupOldAttributionData() } just Runs
+
+        var error: PurchasesError? = null
+        identityManager.logOut { error = it }
+
+        assertThat(error).isNull()
```

Comment:
There should be a check that the completion is called, this will be null by default